neutrons/SNAPRed
# Fixes for beta_d equation issue (EWM 3853)

## Description of work
[Defect 5853](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=3583) describes a simple equation fix within the `DetectorPeakPredictor` algorithm


## Explanation of work
This was a simple one line solution:
```python
beta_d = self.BETA_D_COEFFICIENT * L * np.sin(tTheta / 2) * beta_T
```

to

```python
beta_d = self.BETA_D_COEFFICIENT * L * (np.sin(tTheta / 2) ** -1) * beta_T
```


## To test
Please run `pytest` and ensure all tests pass. One json file containing data had to be modified to fix the failing tests.
### Dev testing



### CIS testing
No CIS script applicable here. Test to insure the outcome of this equation is correct. 


``` python
# N/A
```



## Link to EWM item


[EWM#3583](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=3583)
